example/README.md

```diff
@@ -53,7 +53,7 @@ Test data comes from the @cumulus/test-data package and is uploaded to S3 during
 
 Tests are written and run with [jasmine](https://jasmine.github.io/setup/nodejs.html).
 
-To run all of the tests, run `DEPLOYMENT=<name-of-your-deployment> npm test` in the top level of the repository.
+To run all of the tests, run `DEPLOYMENT=<name-of-your-deployment> npm test` in this directory.
```

Comment:
It seems like this documentation is referring to the integration tests, so I think this change is correct. But feel free to tell me if I'm wrong.

Reply:
👍